src/plugins/dashboard/public/application/embeddable/grid/_dashboard_direct_query_sync.scss

```diff
@@ -0,0 +1,4 @@
+.dshDashboardGrid__syncBar {
```

Comment:
this is missing license header

Reply:
I was following the existing file format such as: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/70ab5be42f77611576a11f519b1433e2c759f83e/src/plugins/dashboard/public/application/embeddable/viewport/_dashboard_viewport.scss#L1